Tasks/WindowsMachineFileCopy/task.loc.json

```diff
@@ -55,12 +55,12 @@
       }
     },
     {
-      "name": "MachineFilter",
+      "name": "MachineNames",
       "type": "string",
-      "label": "ms-resource:loc.input.label.MachineFilter",
+      "label": "ms-resource:loc.input.label.MachineNames",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "ms-resource:loc.input.help.MachineFilter"
+      "helpMarkDown": "ms-resource:loc.input.help.MachineNames"
```

Comment:
Is the roles/filter the only change? Are there any other changes done between M83 and M84 that has modified "name" field for any Task?